modules/broker-amqp/src/main/resources/swagger.yaml

```diff
@@ -34,6 +34,40 @@ paths:
               type: array
               items:
                 $ref: '#/definitions/ConnectionMetadata'
+          '401':
+            description: Authentication Data is missing or invalid
+            headers:
+              WWW_Authenticate:
+                type: string
+            schema:
+              $ref: '#/definitions/Error'
+  /connections/{id}:
+      parameters:
+        - in: path
+          name: id
+          type: integer
+          required: true
+          description: Identifier of the connection
+      delete:
+        operationId: closeConnection
+        summary: Force disconnect the specified connection.
+        description: >-
+          Disconnects the specified amqp connection if the connection exists in the broker
+        produces:
+          - application/json
+        responses:
+          '202':
+            description: Connection removal request submitted.
+            schema:
+              $ref: '#/definitions/ConnectionCloseResponse'
+          '400':
+            description: Bad request. Invalid request or validation error.
+            schema:
+              $ref: '#/definitions/Error'
+          '404':
+            description: Connection not found
+            schema:
+              $ref: '#/definitions/Error'
```

Comment:
Shall we add 403 response as well? Also can you raise an issue for missing 403 response for broker core swagger as well?